### ISSUE TITLE
Zeromail: Internal error: AttributeError: /usr/lib/x86_64-linux-gnu/libssl.so.1.1: undefined symbol: ECDH_OpenSSL

### ISSUE THREAD

**Issue by VerySmallSnake** [HelloZeroNet/ZeroNet#2092]
### Step 1: Please describe your environment

  * ZeroNet version: last
  * Operating system: whonix
  * Web browser: tor
  * Tor status: always

### Step 2: Describe the problem:

Opening ZeroMail get this mesage:
Internal error: AttributeError: /usr/lib/x86_64-linux-gnu/libssl.so.1.1: undefined symbol: ECDH_OpenSSL
UiWebsocket.py line 85 > UiWebsocket.py line 234 > CryptMessage/CryptMessagePlugin.py line 86 > __init__.py line 43 > openssl.py line 513 > openssl.py line 168 > __init__.py line 369 > __init__.py line 374

I also get warning about time sync: Looks like your system time is out of sync. Other users may not see your posted content and other problems could happen. How to fix them?

**Comment by VerySmallSnake** [comment]
I tried to restart the service and send a message, but all English symbols now turned into something strange. Internal error: AttributeError: module 'pyelliptic' has no attribute 'Cipher'UiWebsocket.py line 85 > UiWebsocket.py line 234 > CryptMessage/CryptMessagePlugin.py line 73

And then I also get SSL error

**Comment by HelloZeroNet** [comment]
Right now that plugin only supports openssl 1.0.x, so you need to install `apt install openssl1.0` (or similar with whonix package manager)

As for the time error: is your system time in sync?
What do you see for `Time correction: Xs` on page http://127.0.0.1:43110/Stats ? (end of first line)

**Comment by VerySmallSnake** [comment]
There is no such version in the package:
```
(zeronet) user@host:~/Desktop/ZeroNet-py3$ apt-cache showpkg openssl
Package: openssl
Versions: 
1.1.1c-1 (/var/lib/apt/lists/deb.debian.org_debian_dists_buster_main_binary-amd64_Packages) (/var/lib/dpkg/status)
```

I tried to install openssl1.0 from its site, even deleted old usr/bin/openssl and installed it there, but /usr/lib/x86_64-linux-gnu/libssl.so.1.1 is still working. 

I managed to fix time, Time correction: 0.96s, it was about 7 hours

**Comment by HelloZeroNet** [comment]
OpenSSL 1.1 is supported now by using a modified version of pyellptic library: https://github.com/HelloZeroNet/ZeroNet/commit/fbafd231770b1892ff771f28c4534eec5ef1b35d
